phpstan.neon

```diff
@@ -114,6 +114,7 @@ parameters:
                 - rules/DeadCode/NodeManipulator/LivingCodeManipulator.php
                 - rules/EarlyReturn/Rector/If_/ChangeAndIfToEarlyReturnRector.php
                 - src/PhpParser/NodeTransformer.php
+                - src/FileSystem/FilesFinder.php
```

Comment:
Need to add this exception here because now the `findInDirectoriesAndFiles` returns an array by unpacking two arrays and this is incorrectly seen as wrong